src/lib.rs

```diff
@@ -131,9 +139,9 @@ pub fn jitter<P: AsRef<Path>, W: Write>(
                 // Never treat the origin/destination key as numeric
                 Value::String(value)
             } else if let Ok(x) = value.parse::<f64>() {
-                // Scale all of the numeric values
-                // TODO Crashes on NaNs, infinity
-                Value::Number(serde_json::Number::from_f64(x / repeat).unwrap())
+                // Scale all of the numeric values, unless all_key for this row is 0
+                let scaled = if repeat == 0.0 { x } else { x / repeat };
```

Comment:
Here's part of the problem we discussed yesterday. We only transform numeric values if `all_key / max_per_od` is non-zero now.

Reply:
Sounds reasonable. In my mind `repeat` would be 1 when the `threshold_key` is 0 but this makes that explicit (`x / 1` equals `x`).

Reply:
Looking at other commits and comments, maybe `repeat` could be set as 1 or more on creation (not zero). But that does matter so ignore these comments!

Reply:
https://github.com/dabreegster/odjitter/commit/6f42425defc44ad4e57b8b0051d0fad11c6a023b
You're right, setting to 1 is simpler. Thanks!

Reply:
Awesome! Feel I'm taking baby steps in understanding Rust code that will make me a better programmer all round. Many thanks for quick implementation and glad the comment led to a positive change in the code base :pray: